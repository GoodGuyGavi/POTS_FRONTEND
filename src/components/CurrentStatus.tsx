import React from "react";
import { Card, Feed, Label } from "semantic-ui-react";

export interface IStatusCurrentStatusProps {}

const CurrentStatus = (props: IStatusCurrentStatusProps) => {
  return (
    <Card.Content>
      <Label style={{ background: "black", color: "white" }}>
        <Card.Header style={{ paddingRight: 90, margin: 5, fontSize: 20 }}>
          Current Status
        </Card.Header>
      </Label>
    </Card.Content>
  );
};

export default CurrentStatus;
