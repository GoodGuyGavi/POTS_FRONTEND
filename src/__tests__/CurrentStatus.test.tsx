/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
// import ShallowRenderer from "react-test-renderer/shallow";

// // Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";

import CurrentStatus from "../components/CurrentStatus";

const props = {
  title: "Title"
};

describe("Current Status", () => {
  test("renders correctly", () => {
    expect(shallow(<CurrentStatus {...props} />)).toMatchSnapshot();
  });
});
