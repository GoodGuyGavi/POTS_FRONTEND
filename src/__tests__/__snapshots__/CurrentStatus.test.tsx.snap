// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current Status renders correctly 1`] = `ShallowWrapper {}`;
